aries-site/src/pages/components/tabs.mdx

```diff
@@ -121,3 +115,13 @@ An individual Tab can take on various states: enabled, hover, and active. The st
 >
   <TabStatesExample />
 </Example>
+
+### Accessibility
```

Comment:
Wring heading level here, Accessibility section shouldn't appear under the variants section